website/pages/docs/guide/enums.mdx

```diff
@@ -71,6 +71,44 @@ Enums can be defined a number of different ways:
    Using a values object like this enables defining additional options like a description for each
    enum value.
 
+1. Using an object with `as const`
```

Comment:
Maybe something along the lines of use the values of an existing object type, and similarly mentioned using the keys of the of the object instead.

In general the current docs don't clearly explain the differences between enum names and values and why you may or may not want them to be the same.   I think these examples blur that line a bit, and I want to be explicit here that this will be using the key or value as both name and value.


I think the most useful pattern is actually to go from `{ Foo: 'foo' }` -> `{ Foo: { value: 'foo' } }`, because internally you probably want to use `return MyType.foo`, and in the schema see `enum MyType { Foo }`